posthog/api/team.py

```diff
@@ -34,26 +34,34 @@ class PremiumMultiprojectPermissions(permissions.BasePermission):
 
     def has_permission(self, request: request.Request, view) -> bool:
         user = cast(User, request.user)
-        if request.method in CREATE_METHODS and (
-            (user.organization is None)
-            or (
-                # if we're not requesting to make a demo project
-                # and if the org already has more than 1 non-demo project (need to be able to make the initial project)
-                # and the org isn't allowed to make multiple projects
-                "is_demo" not in request.data
```

Comment:
Was the problem here that if `request.data["is_demo"] === False`, that `"is_demo" not in request.data` would evaluate to `False` (because `"is_demo"` IS in `request.data`, even if it is set to False?

Reply:
correct

Reply:
Ok good to know. Thanks!